internal/plugin/reflike.go

```diff
@@ -28,20 +18,12 @@ func parseReflike(s, projectDir string) (Includable, error) {
 	if err != nil {
 		return nil, err
 	}
-	reflike := RefLike{ref, pluginConfigName, s}
 	switch ref.Type {
 	case flake.TypePath:
-		return newLocalPlugin(reflike, projectDir)
+		return newLocalPlugin(ref, projectDir)
```

Comment:
Could this function and file be renamed to `parseIncludable` and `includable.go`?